README.md

```diff
@@ -79,12 +72,21 @@ For more details on how to use `genezio`, you can check out the official [docume
 
 # Features
 
--   👩‍💻&nbsp; Write your backend code in Javascript, Typescript or Dart.
--   🪛&nbsp; Deploy your backend in no-time on a serverless infrastructure.
--   🖼️&nbsp; Host your frontend on the genezio infrastructure.
--   🪄&nbsp; A magically-generated SDK to call your server functions from the client (supported languages: JS, TS, Swift, Python, Dart).
--   👀&nbsp; A handful of examples to start from at [genezio-examples](https://github.com/Genez-io/genezio-examples/).
--   🔨&nbsp; A dedicated GitHub Action to integrate in your CI/CD at [genezio-github-action](https://github.com/Genez-io/genezio-github-action).
+-   💡 Full static type-safety with **auto-completion** in your favorite editor.
+-   🧩 **Typesafe RPC**: Implemented through parsing the backend code and storing the AST, this allows for generating the SDK in any frontend language
```

Comment:
AST - this is a technicality. Focus on the fact that we do type safety and auto-complete in your IDE on top of multiple backend programming languages like TypeScript, Go, etc